### ISSUE TITLE
This should  be merged into official homebrew

### ISSUE THREAD

**Issue by DarthMDev** [derekkwok/homebrew-python#1]
Thoughts?

**Comment by derekkwok** [comment]
Do you have any particular reason you feel that this should be merged into official homebrew?

Homebrew already provides the `python` and `python3` formula by default, which is what most people use. Also `brew tap` functionality is relatively easy to use, so I don't see a really strong reason to merge something like this into homebrew.

Also, I don't work on homebrew, so any new formula that you want added should be directed to the homebrew project instead.

**Comment by EmilStenstrom** [comment]
I agree that this should be in official homebrew. 3.5 and 3.6 is out, and tapping a second repository for that is a little bit annoying. If your code was in official homebrew someone else could help out!

**Comment by gwynnebaer** [comment]
At minimum, let's add 3.5 to the tap?
[python35.rb.zip](https://github.com/derekkwok/homebrew-python/files/712330/python35.rb.zip)
